Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -359,10 +359,235 @@ C_130J:defineTumb("AUG_ICS_TRANSMISSION_SELECTOR", devices.VOLUME_MANAGER, 3144,
+C_130J:definePushButton("CPLT_CNI_KBD_H", devices.C_CNI, 3049, 1217, CPLT_CNI_MU, "Copilot CNI-MU H Key")
+C_130J:definePushButton("CPLT_CNI_KBD_I", devices.C_CNI, 3050, 1218, CPLT_CNI_MU, "Copilot CNI-MU I Key")
+C_130J:definePushButton("CPLT_CNI_KBD_J", devices.C_CNI, 3051, 1219, CPLT_CNI_MU, "Copilot CNI-MU J Key")
+C_130J:definePushButton("CPLT_CNI_KBD_K", devices.C_CNI, 3052, 1220, CPLT_CNI_MU, "Copilot CNI-MU K Key")
+C_130J:definePushButton("CPLT_CNI_KBD_L", devices.C_CNI, 3053, 1221, CPLT_CNI_MU, "Copilot CNI-MU L Key")
+C_130J:definePushButton("CPLT_CNI_KBD_M", devices.C_CNI, 3054, 1222, CPLT_CNI_MU, "Copilot CNI-MU M Key")
+C_130J:definePushButton("CPLT_CNI_KBD_N", devices.C_CNI, 3055, 1223, CPLT_CNI_MU, "Copilot CNI-MU N Key")
+C_130J:definePushButton("CPLT_CNI_KBD_O", devices.C_CNI, 3056, 1224, CPLT_CNI_MU, "Copilot CNI-MU O Key")
+C_130J:definePushButton("CPLT_CNI_KBD_P", devices.C_CNI, 3057, 1225, CPLT_CNI_MU, "Copilot CNI-MU P Key")
+C_130J:definePushButton("CPLT_CNI_KBD_Q", devices.C_CNI, 3058, 1226, CPLT_CNI_MU, "Copilot CNI-MU Q Key")
+C_130J:definePushButton("CPLT_CNI_KBD_R", devices.C_CNI, 3059, 1227, CPLT_CNI_MU, "Copilot CNI-MU R Key")
+C_130J:definePushButton("CPLT_CNI_KBD_S", devices.C_CNI, 3060, 1228, CPLT_CNI_MU, "Copilot CNI-MU S Key")
+C_130J:definePushButton("CPLT_CNI_KBD_T", devices.C_CNI, 3061, 1229, CPLT_CNI_MU, "Copilot CNI-MU T Key")
+C_130J:definePushButton("CPLT_CNI_KBD_U", devices.C_CNI, 3062, 1230, CPLT_CNI_MU, "Copilot CNI-MU U Key")
+C_130J:definePushButton("CPLT_CNI_KBD_V", devices.C_CNI, 3063, 1231, CPLT_CNI_MU, "Copilot CNI-MU V Key")
+C_130J:definePushButton("CPLT_CNI_KBD_W", devices.C_CNI, 3064, 1232, CPLT_CNI_MU, "Copilot CNI-MU W Key")
+C_130J:definePushButton("CPLT_CNI_KBD_X", devices.C_CNI, 3065, 1233, CPLT_CNI_MU, "Copilot CNI-MU X Key")
+C_130J:definePushButton("CPLT_CNI_KBD_Y", devices.C_CNI, 3066, 1234, CPLT_CNI_MU, "Copilot CNI-MU Y Key")
+C_130J:definePushButton("CPLT_CNI_KBD_Z", devices.C_CNI, 3067, 1235, CPLT_CNI_MU, "Copilot CNI-MU Z Key")
+C_130J:definePushButton("CPLT_CNI_KBD_SLASH", devices.C_CNI, 3068, 1238, CPLT_CNI_MU, "Copilot CNI-MU / Key")
+C_130J:definePushButton("CPLT_CNI_KBD_DEL", devices.C_CNI, 3028, 1237, CPLT_CNI_MU, "Copilot CNI-MU DEL Key")
+C_130J:definePushButton("CPLT_CNI_KBD_CLR", devices.C_CNI, 3029, 1239, CPLT_CNI_MU, "Copilot CNI-MU CLR Key")
+C_130J:defineIndicatorLight("CPLT_CNI_DSPY_LED", 4141, CPLT_CNI_MU, "Copilot CNI-MU DSPY Light (Green)")
+C_130J:defineIndicatorLight("CPLT_CNI_MSG_LED", 4142, CPLT_CNI_MU, "Copilot CNI-MU MSG Light (Green)")
+C_130J:defineIndicatorLight("CPLT_CNI_FAIL_LED", 4143, CPLT_CNI_MU, "Copilot CNI-MU FAIL Light (Yellow)")
+C_130J:defineIndicatorLight("CPLT_CNI_OFSET_LED", 4144, CPLT_CNI_MU, "Copilot CNI-MU OFSET Light (Green)")
+C_130J:defineIndicatorLight("CPLT_CNI_EXEC_LED", 3392, CPLT_CNI_MU, "Copilot CNI-MU EXEC Light (Green)")
```

Comment:
This should be a defineFloat as the light is dimmable and it's brightness is linked to this draw arg.